packages/bitcore-node/src/modules/ethereum/models/transaction.ts

```diff
@@ -108,14 +124,19 @@ export class EthTransactionModel extends BaseTransaction<IEthTransaction> {
 
   async expireBalanceCache(txOps: Array<any>) {
     for (const op of txOps) {
-      let batch = new Array<{ tokenAddress?: string; address: string }>();
+      let batch = new Array<{ multisigContractAdress?: string; tokenAddress?: string; address: string }>();
       const { chain, network } = op.updateOne.filter;
       const { from, to, abiType } = op.updateOne.update.$set;
       batch = batch.concat([{ address: from }, { address: to }]);
-      if (abiType && abiType.params.length) {
+      if (abiType && abiType.type === 'ERC20' && abiType.params.length) {
         batch.push({ address: from, tokenAddress: to });
         batch.push({ address: abiType.params[0].value, tokenAddress: to });
       }
+
```

Comment:
we should also push a batch for the internal transfers here

Reply:
Done. Thanks !